include/onnxruntime/core/graph/graph.h

```diff
@@ -80,6 +80,19 @@ class Node {
   };
 
   explicit Node() = default;
+
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
+  Node(const std::string& name,
```

Comment:
> std::string& name, [](http://example.com/codeflow?start=12&length=19)

std::string_view by value for all strings.

Reply:
The constructor added here is just to bridge between a caller and Node::init(const std::string&...).

Reply:
Switched to string_view, thanks!